File: SuperiorPackGroup/Code Classes/ItemTransactionSummary.vb
Public Class ItemTransactionSummary

    ''' <summary>
    ''' Initializes a new instance of the ItemTransactionSummary class.
    ''' </summary>
    Public Sub New(ByVal customerName As String, ByVal itemCode As String, ByVal itemDescription As String, ByVal itemType As String, ByVal produced As Double, ByVal usedInProduction As Double, ByVal shipped As Double, ByVal received As Double,
                   ByVal adjusted As Double)

        Me.CustomerName = customerName
        Me.ItemCode = itemCode
        Me.ItemDescription = itemDescription
        Me.ItemType = itemType
        Me.Produced = produced
        Me.UsedInProduction = usedInProduction
        Me.Shipped = shipped
        Me.Received = received
        Me.Adjusted = adjusted

    End Sub

    Public Sub New()

    End Sub

    Public Property CustomerName() As String
    Public Property ItemCode() As String
    Public Property ItemDescription() As String
    Public Property ItemType() As String
    Public Property Produced() As Double
    Public Property UsedInProduction() As Double
    Public Property Shipped() As Double
    Public Property Received() As Double
    Public Property Adjusted() As Double

End Class

File: SuperiorPackGroup/Code Classes/ItemTransactionSummary.cs
//INSTANT C# NOTE: Formerly VB project-level imports:
using System;
using System.Collections;
using System.Collections.Generic;
using System.Data;
using System.Drawing;
using System.Diagnostics;
using System.Windows.Forms;

namespace SuperiorPackGroup
{
	public class ItemTransactionSummary
	{

		/// <summary>
		/// Initializes a new instance of the ItemTransactionSummary class.
		/// </summary>
		public ItemTransactionSummary(string customerName, string itemCode, string itemDescription, string itemType, double produced, double usedInProduction, double shipped, double received, double adjusted)
		{

			CustomerName = customerName;
			ItemCode = itemCode;
			ItemDescription = itemDescription;
			ItemType = itemType;
			Produced = produced;
			UsedInProduction = usedInProduction;
			Shipped = shipped;
			Received = received;
			Adjusted = adjusted;

		}

		public ItemTransactionSummary()
		{

		}

		public string CustomerName {get; set;}
		public string ItemCode {get; set;}
		public string ItemDescription {get; set;}
		public string ItemType {get; set;}
		public double Produced {get; set;}
		public double UsedInProduction {get; set;}
		public double Shipped {get; set;}
		public double Received {get; set;}
		public double Adjusted {get; set;}

	}

}